docs/onboarding.md

```diff
@@ -0,0 +1,40 @@
+# Dataland QA Lab Onboarding Guide
+
+Please follow the steps below to get started with the Dataland QA Lab project.
+
+## Required Software
+Please ensure the following software is installed on your machine:
+* [Git](https://git-scm.com/downloads)
+* [Python 3.12](https://www.python.org/downloads/)
+* [PDM](https://pdm-project.org/en/latest/#installation) - a Python package manager
+* A recent version of Java (on windows, we recommend [Eclipse Temurin](https://adoptium.net/de/temurin/releases/) 21 for easy installation. On linux, just use your package manager ;))
+* An IDE of your choice (we recommend [VS Code](https://code.visualstudio.com/download). Alternatively you may use [IntelliJ Ultimate/PyCharm](https://www.jetbrains.com/de-de/pycharm/download/))
+
```

Comment:
Let's insist on VS Code and also list recommended VS Code Extensions:

- Jupyter
- Python
- Git Blame
- Git Graph
- GitHub Pull Requests
- Python Debugger
- Remote - SSH
- Ruff
- vscode-pdf

Reply:
Done. I additionally added these extensions as workspace recommendations. That ensures that you can install all of them at once when you open the project.

Reply:
Also: I added some settings in the settings.json for pytest support and such